sherpa/astro/tests/test_astro_data.py

```diff
@@ -70,7 +70,7 @@ class test_filter_energy_grid(SherpaTestCase):
         2.71560001e+00, 2.86159992e+00, 3.08060002e+00, 3.38720012e+00,
         3.56240010e+00, 3.79600000e+00, 4.02960014e+00, 4.24860001e+00,
         4.71579981e+00, 5.02239990e+00, 5.37279987e+00, 5.89839983e+00,
-        6.57000017e+00, 9.86960030e+00], np.float)
+        6.57000017e+00, 9.86960030e+00], float)
```

Comment:
This is fine, but there is no need to even have the "float" here. `np.array` defaults to float anyway.